test/test_jit_cuda_fuser.py

```diff
@@ -3198,6 +3204,33 @@ def test_register_fuser(self):
         self.assertTrue(torch._C._jit_set_nvfuser_enabled(False))
         self.assertFalse(torch._C._jit_nvfuser_enabled())
 
+_tracing_ops = partial(ops, dtypes=OpDTypes.supported,
+                       allowed_dtypes=(torch.float, torch.cfloat))
+
```

Comment:
is there a reason we're restricting to float and cfloat ?

Reply:
copied this from the [variant_consistency tests](https://github.com/pytorch/pytorch/blob/master/test/test_ops.py#L32), where 

```python
# variant testing is only done with torch.float and torch.cfloat to avoid
#   excessive test times and maximize signal to noise ratio
```

Reply:
there's a slow test that runs nightly, we could at least run it then (and initially, to flush out issue)

Reply:
ctrl-f SLOW_TEST or something and you'll find it